programs/memo/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+// Last updated at mainnet-beta slot height: 298485997
+#[cfg(feature = "memo")]
+pub mod memo;
```

Comment:
Same here on the doc type please.

```suggestion
//! Last updated at mainnet-beta slot height: 298485997

#[cfg(feature = "memo")]
pub mod memo;
```

Reply:
Also will have to just adjust the scripts a bit.